src/main.cc

```diff
@@ -10,43 +10,45 @@
 namespace po = boost::program_options;
 
 std::string version_number() {
-    std::stringstream ss;
-    ss << Hopper_VERSION_MAJOR << '.' << Hopper_VERSION_MINOR;
-    return ss.str();
+  std::stringstream ss;
+  ss << Hopper_VERSION_MAJOR << '.' << Hopper_VERSION_MINOR;
+  return ss.str();
 }
 
 po::variables_map parse_input(int argc, char *argv[]) {
-    // Declare the supported options.
-    po::options_description desc("Allowed options");
-    desc.add_options()
-        ("help", "produce help message")
-        ("input-files", po::value<std::vector<std::string>>(), "Input files")
-        ;
-    po::positional_options_description p;
-    p.add("input-files", -1);
-
-    po::variables_map vm;
-    po::store(po::command_line_parser(argc, argv).
-              options(desc).positional(p).run(), vm);
-    po::notify(vm);
-
-    if (vm.count("help") || !vm.count("input-files")) {
-        std::cout << "gracc Hopper version " << version_number() << "\n"
-                  << desc << "\n";
-        exit(1);
-    }
-    return vm;
+  // Declare the supported options.
+  po::options_description desc("Allowed options");
+  desc.add_options()
+    ("help", "produce help message")
+    ("input-files", po::value<std::vector<std::string>>(), "Input files")
+    ;
+  po::positional_options_description p;
+  p.add("input-files", -1);
+
+  po::variables_map vm;
+  po::store(po::command_line_parser(argc, argv).
+            options(desc).positional(p).run(), vm);
+  po::notify(vm);
+
+  if (vm.count("help") || !vm.count("input-files")) {
+    std::cout << "gracc Hopper version " << version_number() << "\n"
+      << desc << "\n";
+    exit(1);
```

Comment:
That's a matter of convention, many programs return non-0 when displaying help, it helps to show that it didn't do anything.